log_pytest/tasks.py

```diff
@@ -0,0 +1,17 @@
+import sys
+from pathlib import Path
+
+# Add the devutils even if the poetry env isn't setup (to do a 'inv devinstall').
+try:
+    import devutils
+except ImportError:
+    devutils_src = Path(__file__).absolute().parent.parent / "devutils" / "src"
+    assert devutils_src.exists(), f"{devutils_src} does not exist."
+    sys.path.append(str(devutils_src))
+
+if True:  # Create block to stop isort from changing the import below.
```

Comment:
I sort supports comments to control the functionality per-line/block/file: https://pycqa.github.io/isort/docs/configuration/action_comments.html